packages/orm/src/ReactionsManager.ts

```diff
@@ -202,20 +206,11 @@ export class ReactionsManager {
         await this.em.assignNewIds();
         const actions = [...actionsPendingAssignedIds.values()];
         const startTime = this.logger?.now() ?? 0;
-        const results = await Promise.allSettled(
-          actions.filter((a) => !a.entity.isDeletedEntity).map((a) => this.#doAction(a)),
-        );
+        const results = await Promise.allSettled(actions.map((a) => this.#doAction(a)));
         const endTime = this.logger?.now() ?? 0;
         this.logger?.logLoadingTime(this.em, endTime - startTime);
         results.forEach((result) => {
-          if (result.status === "rejected") {
-            if (result.reason instanceof TypeError) {
-              // Defer to the validation error to catch this
-              this.suppressedTypeErrors.push(result.reason);
-            } else {
-              failures.push(result.reason);
-            }
-          }
+          if (result.status === "rejected") failures.push(result.reason);
```

Comment:
What if a RF failed once b/c of an IdError but then fails as 2nd time due to a `TypeError`?